goentri.com.squareup.json

```diff
@@ -3,25 +3,26 @@
     "providerName": "Entri",
     "serviceId": "squareup",
     "serviceName": "Squareup",
-    "version": 1,
+    "version": 2,
     "logoUrl": "https://cdn.goentri.com/logo.svg",
     "description": "Allows user to easily set up domain using Entri",
     "syncPubKeyDomain": "goentri.com",
     "syncRedirectDomain": "api.goentri.com, goentri.com, entri.com, app.entri.com,app.goentri.com",
-    "variableDescription":"ip is the ip to point the domain",
+    "variableDescription":"ip is the ip to point the domain and domain is the domain to be pointed",
+    "hostRequired": false,
     "sharedProviderName": true,
     "records": [
       {
         "type":"A",
         "host":"@",
         "pointsTo":"%ip%",
-        "ttl": 300
+        "ttl": 600
       },
       {
-        "type":"A",
-        "host":"www",
-        "pointsTo":"%ip%",
-        "ttl": 300
+         "type": "CNAME",
+         "host": "www",
+         "pointsTo": "%domain%",
```

Comment:
Hi, you'd rather want to use `@` instead of `%domain%`, otherwise if the template is used with a subdomain for example `host=foo` it will generate `www.foo.example.com` pointing to `example.com` instead of `foo.example.com`.